### PR TITLE
Completed Dataset and Dataloader

### PR DESCRIPTION
**Pull request by KCui0327**
Dataset and dataloader created under ./data_module folder

**Dataset**
- Class created for dataset where it can access labels and tensors of images
- dataset folder created which contains the main data
- have folder dir. manipulation to migrate original data to dataset folder

**Dataloader**
- dataloader created out of dataset folder with PyTorch library

### CODE REVIEW

**Comment by thomaslin2020** [comment]
Please convert the notebooks into python scripts so that we can run through the pipeline of loading data, doing data transformations, and model training easily. 

Please also put model settings settings into a yaml file as it would allow us to iterate much faster on different model settings. You can organize the yaml files by the stage of the pipeline (data loading, data transform, model training, model evaluation, and model inference). Here is a popular [library](https://hydra.cc/docs/intro/) to easily manage configurations.
